configs/exp.yaml

```diff
@@ -0,0 +1,77 @@
+modules:
+  - seacliff
+  - galsim_extra
+
+image:
+  type: RubinCalexp
+
+  nobjects:
+    type: MixedNObjects
+    ngalaxies:
+        type: RandomPoisson
+        mean: 1000
+
+  world_pos:
+    type: RADec
+    type: RADec
+    ra:
+      type: Degrees
+      theta: { type: Random, min: "$ra_min_deg", max: "$ra_max_deg" }
+    dec:
+      type: Radians
+      theta:
+        type: RandomDistribution
+        function: "math.cos(x)"  # Uniform on the sky means P(delta) ~ cos(delta)
+        x_min: "$numpy.radians(dec_min_deg)"
+        x_max: "$numpy.radians(dec_max_deg)"
```

Comment:
This is allowed here, but it's probably better to put this in the `stamp` section.  Where it means the same thing, btw.  

After I made the `stamp` concept, I realized a few things that had been in `image` really belonged in `stamp`, including the positions, but I left them allowed in `image` for backwards compatibility.